PRODUCTIONISATION.md

```diff
@@ -147,3 +152,4 @@ To be applied if infrastructure has been orchestrated and provisioned specifical
 [rfc_012]: https://github.com/madetech/rfcs/blob/master/rfc-012-makefile-standards.md "RFC 012 - Makefile standards"
 [rfc_013]: https://github.com/madetech/rfcs/blob/master/rfc-013-code-repositories.md "RFC 013 - Code repository standards"
 [ruby_reference]: reference/ruby.md
+[adr_reference]: https://github.com/madetech/architecture-decision-log
```

Comment:
This is a private repo.

Can you link to a public version of ADRs please?

Reply:
There is no public repo ATM. I think this repo can be made public or we can update this link when a public repo for ADRs is available(?).

Reply:
Best to link to the blog we reference in our ADR repo until we have a public ADR somewhere then:

https://web.archive.org/web/20190430063238/http://thinkrelevance.com/blog/2011/11/15/documenting-architecture-decisions

Reply:
Resolved in https://github.com/madetech/productionisation/pull/51/commits/8f74a070e487c1c0d8c316bf8bd5f9fecfd10b00.